src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/ampMetadataExtractor/index.test.js

```diff
@@ -1,4 +1,4 @@
-import ampMetadataExtractor from './ampMetadataExtractor';
+import ampMetadataExtractor from '.';
```

Comment:
I'd move logging testing from `index.test.js` to here where it relates to amp includes.